server/models/User.js

```diff
@@ -21,6 +21,24 @@ const userSchema = new mongoose.Schema({
     type: String,
     required: true
   },
+  url: {
+    type: String,
+    required: true,
+  },
```

Comment:
Would making unique be beneficial here? Or was this done intentionally. We wouldn't want the User url to be duplicated in the database for conflicting reasons?